test/server/cards/09.1-FtE/MomentOfPerfectBeauty.spec.js

```diff
@@ -0,0 +1,94 @@
+                        inPlay: ['adept-of-the-waves', 'miya-mystic', 'tattooed-wanderer'],
+                        hand: ['fine-katana', 'hawk-tattoo']
+                    }
+                });
+
+                this.momentofbeauty = this.player1.findCardByName('moment-of-perfect-beauty');
+                this.dojichallenger = this.player1.findCardByName('doji-challenger');
+                this.finekatana = this.player2.findCardByName('fine-katana');
+                this.hawktattoo = this.player2.findCardByName('hawk-tattoo');
+                this.noMoreActions();
+            });
+
+            it('should not play outside of conflict', function() {
+                this.player1.clickCard(this.momentofbeauty);
+                expect(this.momentofbeauty.location).toBe('hand');
+            });
+
+            it('should not play unless controlling more participating honored characters', function() {
+                this.initiateConflict({
+                    attackers: ['doji-challenger'],
+                    defenders: ['adept-of-the-waves'],
+                    type: 'political'
+                });
+                this.player2.pass();
+                this.player1.clickCard(this.momentofbeauty);
+                expect(this.momentofbeauty.location).toBe('hand');
+            });
+
+            it('should play with the right conditions', function() {
+                this.initiateConflict({
+                    attackers: ['doji-challenger'],
+                    defenders: ['adept-of-the-waves'],
+                    type: 'political'
+                });
+                this.dojichallenger.honor();
+                this.player2.pass();
+                this.player1.clickCard(this.momentofbeauty);
+                expect(this.momentofbeauty.location).toBe('conflict discard pile');
+            });
+
+            it('should end conflict if other player passes', function() {
+                this.initiateConflict({
+                    attackers: ['doji-challenger'],
+                    defenders: ['adept-of-the-waves'],
+                    type: 'political'
+                });
+                this.dojichallenger.honor();
+                this.player2.pass();
+                this.player1.clickCard(this.momentofbeauty);
+                this.player2.pass();
+                expect(this.player1).not.toHavePrompt('Conflict Action Window');
+            });
+
+            it('should end conflict if other player plays a card', function() {
+                this.initiateConflict({
+                    attackers: ['doji-challenger'],
+                    defenders: ['adept-of-the-waves'],
+                    type: 'political'
+                });
+                this.dojichallenger.honor();
+                this.player2.pass();
+                this.player1.clickCard(this.momentofbeauty);
+                this.player2.clickCard(this.finekatana);
+                expect(this.player1).not.toHavePrompt('Conflict Action Window');
```

Comment:
This test doesn't work - player2 is currently being prompted to choose a character to play Katana on.  As a general rule, check for the prompt you expect the player to have.